LineApp/src/main/java/org/example/GitHubApiClient.java

```diff
@@ -21,6 +22,7 @@ public GitHubApiClient(String ownerName, String ownerRepos) {
         this.ownerRepos = ownerRepos;

```

Comment:
need checks for constructor argument validity, you don't want to even attempt making REST calls if ownerName or repo are empty